medcat/cdb.py

```diff
@@ -802,3 +802,113 @@ def calculate_hash(self):
         self._hash = hasher.hexdigest()
         logger.info("Found new CDB hash: %s", self._hash)
         return self._hash
+
+    @staticmethod
+    def merge_cdb(cdb1: "CDB", cdb2: "CDB", overwrite: int = 0, vector_import: dict[str, dict[str, np.array]] = {}):
```

Comment:
You want to use `Dict[str, Dict[str, np.ndarray]`.
I.e `Dict` instead of `dict` since we're still supporting python 3.8 which doesn't support the more readable version.
And `np.ndarray` instead of `np.array` since the latter is a method for creating an instance of the former.

Reply:
I've changed it to `np.ndarray` as suggested.

`Dict` doesn't work in python 3.8-3.11 for me. `dict` does.